### PR TITLE
feat: Add missing translations

### PR DESCRIPTION
**Pull request by Alystrasz**
Adds missing Chinese, French, German, Italian and Spanish translation entries.

Thanks to @GeckoEidechse, `Cloud__#0920`, `wolf109909#5291` and `DumbAndre#5622` for their help.

### CODE REVIEW

File: Northstar.Client/mod/resource/northstar_client_localisation_german.txt [hunk 1]
```diff
@@ -16,6 +19,9 @@ Drücke Ja, um zuzustimmen. Du kannst diese Entscheidung jederzeit im Modmenü 
 		"AUTHENTICATION_AGREEMENT" "Authentifizierungs-Einwilligung"
 		"AUTHENTICATION_AGREEMENT_RESTART" "Ein Neustart ist notwendig, um diese Änderung zu übernehmen"
 
+		"DIALOG_AUTHENTICATING_MASTERSERVER" "Authentifizierung mit Master Server"
+		"AUTHENTICATIONAGREEMENT_NO" "Du hast dich entschieden dich nicht mit Northstar zu authentizifieren. Du kannst die Authentifizierungs-Einwilligung im Modmenü ansehen."
```
Comment (uniboi) [inline Northstar.Client/mod/resource/northstar_client_localisation_german.txt:23]:
```suggestion
		"AUTHENTICATIONAGREEMENT_NO" "Du hast dich gegen die Authentifizierung mit Northstar entschieden. Du kannst die Authentifizierungs-Einwilligung im Modmenü ansehen."
		Sounds better
```

File: Northstar.Client/mod/resource/northstar_client_localisation_german.txt [hunk 2]
```diff
@@ -113,6 +123,17 @@ Drücke Ja, um zuzustimmen. Du kannst diese Entscheidung jederzeit im Modmenü 
 		"HIDDEN_KILL_SURVIVORS" "Töte alle Überlebenden"
 		"HIDDEN_FIRST_HIDDEN" "%s1 ist the The Hidden."
 
+		"PL_sns" "Stock und Stein"
+		"PL_sns_lobby" "Stock und Stein Lobby"
+		"PL_sns_desc" "Frei für Alle. Erziele Kills mit Impulsklingen und Exekutierungen um die Punktzahl des Gegners zurückzusetzen"
+		"PL_sns_abbr" "SuS"
```
Comment (uniboi) [inline Northstar.Client/mod/resource/northstar_client_localisation_german.txt:129]: 💀

File: Northstar.Client/mod/resource/northstar_client_localisation_german.txt [hunk 3]
```diff
@@ -266,6 +287,15 @@ Drücke Ja, um zuzustimmen. Du kannst diese Entscheidung jederzeit im Modmenü 
...
 		"TOTAL_SERVERS" "Server: ^C46C6C00%s1"
 
+		// Mods menu
+		"SHOW" "Anzeigen"
+		"SHOW_ALL" "Alle anzeigen"
+		"SHOW_ONLY_ENABLED" "Nur aktivierte"
+		"SHOW_ONLY_DISABLED" "Nur deaktivierte"
+
+		// Maps menu
+		"HIDE_LOCKED" "Verstecke geperrte"
```
Comment (uniboi) [inline Northstar.Client/mod/resource/northstar_client_localisation_german.txt:297]:
```suggestion
		"HIDE_LOCKED" "Verstecke gesperrte"
```

**Review by uniboi — CHANGES_REQUESTED** [review]

File: Northstar.Client/mod/resource/northstar_client_localisation_german.txt (hunk 2 above)
Comment (F1F7Y) [reply Northstar.Client/mod/resource/northstar_client_localisation_german.txt:129]: sus

Comment (Alystrasz) [reply Northstar.Client/mod/resource/northstar_client_localisation_german.txt:129]: oh my

**Comment by GeckoEidechse** [comment]
I assume I can merge this or is there anything you wanna update still? ^^

**Comment by Alystrasz** [comment]
Yep you can merge this, I don't expect other translations to be added.

**Pull request merged**